tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py36}{,-flake8}
+envlist = {py27,py36}{,-flake8}, linting
```

Comment:
```suggestion
envlist = py27, py36, linting
```

Reply:
Hi @sevein did you notice when you run tox with this newer setting then Flake8 doesn't run locally? That's often beneficial to the developer before pushing. I'm not sure though, maybe it's doing something different, or my configuration needs changing?

Reply:
I think that your `tox.ini` is missing the `linting` definition:

```ini
[testenv:linting]
basepython = python3
deps = pre-commit
commands = pre-commit run --all-files --show-diff-on-failure
```

Then:

```
$ tox -e linting
black....................................................................Passed
flake8...................................................................Failed
hookid: flake8
```

Reply:
:+1: that looks better now.